README.md

```diff
@@ -217,6 +217,17 @@ The ViewModifiers are used to customise the look and feel of the BottomSheet.
 - Changing the threshold does not affect whether either option is enabled.
 - The default threshold is 30% (0.3).
 
+`.iPadFloatingSheet(Bool)`: Makes the iPad's bottom sheet be an actual bottom sheet when set to true.
+- The default is `false`.
+
+`.relativeSheetWidth(Double?)`: Sets the relative width of the sheet.
```

Comment:
I think something similar to the `BottomSheetPosition` would be great here. I can imagine, that soon requests will pop up, that people want to use absolute values. 

**Proposed change:** create a `BottomSheetWidth` and add the cases `.default`, `.relative` and `.absolute`. Then change the current modifier to take one of these enum cases as input. I think a `.dynamic` case would also be great, but honesty I had so much trouble implementing it for the height, I understand if you don’t want to deal with it.

Also, if we add this `BottomSheetWidth`, what do you think of renaming the `BottomSheetPosition` to `BottomSheetHeight`? I’m interested in your opinion.

Reply:
Upgrading the width modifier to accept an enum sounds like a good idea! Although I'll leave implementing `.dynamic` up to someone else, since I have no use case for it (and - as you said - can be a pain to implement).

Regarding the rename of `position` to `height`, that would be a breaking change, so... maybe save that one for a PR with more breaking changes? I'd like to not break anything (for now) 😊

Reply:
Yes that’s what I thought. Thanks for your feedback 👍